Header header

uint16 channel_1
uint16 channel_2
uint16 channel_3
uint16 channel_4
uint16 channel_5
uint16 channel_6
uint16 channel_7
uint16 channel_8
